src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.java

```diff
@@ -607,7 +607,7 @@ public void getVisitsWithVisitorsWithPageAndSize() {
     @Test
     public void getVisitsWithVisitorsWithPageAndSizeAsMax() {
         var page = Integer.MAX_VALUE;
-        var size = Integer.MAX_VALUE;
+        var size = 1;
```

Comment:
this was odd - page size of max * page of max is `MAX_VALUE` * `MAX_VALUE` which blows an int storage and gave us now a different error.  I've therefore changed size to `1` which still fits into the `MAX_VALUE` bound and endpoint now succeeds